### PR TITLE
fix: popup height and scrolling

### LINKED ISSUE #2398: ️🛠 - Fix popup template
### Task description

In case there are insufficient funds + enough tokens, the `SelectToken.view` becomes too tall.

image

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary

closes #2398 

### Platforms

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [x] Windows

## Checklist

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [x] I have verified that new and existing unit tests pass locally with my changes
-   [x] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/popup/PopupTemplate.svelte [hunk 1]
```diff
@@ -91,7 +93,7 @@
                         disabled={busy || _backButton.disabled}
                         width="full"
                         form={_backButton.type === 'submit' ? _backButton.form : undefined}
-                        on:click={_backButton.type === 'button' && _backButton.onClick}
+                        on:click={_backButton.type === 'button' ? _backButton.onClick : undefined}
```
Comment (Tuditi) [inline packages/desktop/components/popup/PopupTemplate.svelte:96]: Why did you change this?

File: packages/desktop/components/popup/Popup.svelte [hunk 2]
```diff
@@ -226,6 +226,7 @@
         bind:this={popupContent}
         class:relative
         class="popup {size}"
+        class:-mt-7={IS_WINDOWS}
```
Comment (Tuditi) [inline packages/desktop/components/popup/Popup.svelte:229]: Doesn't this counteract the if else on l.216-218?

**Review by Tuditi — COMMENTED** [review]
Left some questions :relaxed:

File: packages/desktop/components/popup/PopupTemplate.svelte (hunk 1 above)
Comment (jeeanribeiro) [reply packages/desktop/components/popup/PopupTemplate.svelte:96]: on:click expects an event handler function or undefined but this expression can return the first statement which is a boolean. Also, it was highlighting an error in my VSCode.

**Review by Tuditi — APPROVED** [review]

**Pull request merged**